core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -187,16 +188,20 @@ private void createZkNamespace() {
     String schemaRegistryNamespace = "/" + schemaRegistryZkNamespace;
     schemaRegistryZkUrl = zkConnForNamespaceCreation + schemaRegistryNamespace;
 
-    ZkClient zkClientForNamespaceCreation = new ZkClient(zkConnForNamespaceCreation,
-                                                         zkSessionTimeoutMs, zkSessionTimeoutMs,
-                                                         new ZkStringSerializer());
+    ZkUtils zkUtilsForNamespaceCreation = ZkUtils.apply(
+        zkConnForNamespaceCreation,
+        zkSessionTimeoutMs, zkSessionTimeoutMs,
+        JaasUtils.isZkSecurityEnabled(System.getProperty(JaasUtils.JAVA_LOGIN_CONFIG_PARAM)));
     // create the zookeeper namespace using cluster.name if it doesn't already exist
-    ZkUtils.makeSurePersistentPathExists(zkClientForNamespaceCreation, schemaRegistryNamespace);
+    zkUtilsForNamespaceCreation.makeSurePersistentPathExists(
```

Comment:
@ewencp @fpj  It is configurable using `schema.registry.zk.namespace`

@fpj Is it not the role of a client to create a namespace for its own use? In this case, `schema_registry` by default